client/src/index.js

```diff
@@ -155,39 +156,56 @@ export default function Login() {
     }
 
     const handleSubmit = () => {
+        //Login
         if (state.isLogin) {
-            handleLogin()
-            return
-        }
```

Comment:
Can you revert this to have the `return` statement after the `handleLogin()` function? If we don't have it there, it will also call the `handleSignup()` function